### PR TITLE
style edits on Infinispan blog post

### PR DESCRIPTION
**Pull request by lauracowen**

### CODE REVIEW

File: drafts/2019-12-06-Infinispan-support-with-OpenShift.adoc [hunk 1]
```diff
@@ -108,9 +113,11 @@ Next, you'll need to create the `infinispan.xml` referenced from the `server.xml
...
+The JGroups stack is what determines how your Infinispan servers forms a cluster.  We're using the default Kubernetes template because we're running in OpenShift. See link:https://infinispan.org/docs/dev/titles/configuring/configuring.html#cluster_transport[Configuring Infinispan 10] for Infinispan's complete documentation on setting up your cluster.
 
-```code
+Next, you'll need to create a headless Kubernetes service to enable the Kubernetes JGroups transport stack to form a cluster. The key here is for the `name` of the `selector` to match one of the labels associated with your Open Liberty applications running in OpenShift. For example, an application defined using the command `oc new-app --image-stream=ol-runtime-infinispan-embedded:1.0.0 --name=embedded-servera -l name=ol-runtime-infinispan-embedded` has a label of `name=ol-runtime-infinispan-embedded`. This label  then matches the service defined below, and triggers the application to be a part of the service.
```
Comment (nmittles) [inline drafts/2019-12-06-Infinispan-support-with-OpenShift.adoc]: Double space between `label  then`.

File: drafts/2019-12-06-Infinispan-support-with-OpenShift.adoc [hunk 2]
```diff
@@ -52,25 +51,28 @@ For configuring client/server mode in Open Liberty, we'll start with a sample `s
...
 
-Finally, we'll show you how to start an Infinispan server in your OpenShift environment. In the example below `--name=infinispan-server` maps to `infinispan.client.hotrod.server_list="infinispan-server:11222"` in your Open Liberty server.xml (11222 is the default port). Additionally, `USER` and `PASS` map to `infinispan.client.hotrod.auth_username` and `infinispan.client.hotrod.auth_password` respectively.
+Finally, start an Infinispan server in your OpenShift environment with the following command:
 
-```code
+```
 oc new-app --docker-image=infinispan/server --name=infinispan-server -e USER="sampleUser" -e PASS="samplePassword"
 ```
 
+ In the example below `--name=infinispan-server` maps to `infinispan.client.hotrod.server_list="infinispan-server:11222"` in your Open Liberty server.xml (11222 is the default port). Additionally, `USER` and `PASS` map to `infinispan.client.hotrod.auth_username` and `infinispan.client.hotrod.auth_password` respectively.
```
Comment (nmittles) [inline drafts/2019-12-06-Infinispan-support-with-OpenShift.adoc]: We should say the 'example above`.

File: drafts/2019-12-06-Infinispan-support-with-OpenShift.adoc (hunk 1 above)
Comment (lauracowen) [reply drafts/2019-12-06-Infinispan-support-with-OpenShift.adoc]: done

File: drafts/2019-12-06-Infinispan-support-with-OpenShift.adoc (hunk 2 above)
Comment (lauracowen) [reply drafts/2019-12-06-Infinispan-support-with-OpenShift.adoc]: done

**Comment by lauracowen** [comment]
@nmittles Fixed. Also renamed the file so it's now showing as two files in the 'Files changed' tab. The second is the final version ready to publish.

**Pull request merged**